lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -6,6 +6,19 @@
 
 include_once 'dosomething_global.features.inc';
 define('GLOBAL_ADMIN_ROLES', 'mexico admin,brazil admin');
+define('GLOBAL_ADMIN_LANGUANGES', serialize(['es-mx' => 'mexico admin', 'pt-br' => 'brazil admin']));
```

Comment:
I'm sort of blocked on this being implemented in (for re-directs) https://github.com/DoSomething/phoenix/issues/5020

I can hardcode the values for now and make it dynamic later when we add in the strongarm